app/code/Magento/Shipping/Model/Carrier/AbstractCarrierInterface.php

```diff
@@ -90,7 +90,7 @@ public function checkAvailableShipCountries(\Magento\Framework\DataObject $reque
      * @return $this|\Magento\Framework\DataObject|boolean
      * @api
      */
-    public function proccessAdditionalValidation(\Magento\Framework\DataObject $request);
+    public function processAdditionalValidation(\Magento\Framework\DataObject $request);
```

Comment:
Unfortunately, we are not able to change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)